R/get_confidence_interval.R

```diff
@@ -63,6 +74,34 @@
 #'     # Using the standard error method
 #'     type = "se"
 #'   )
+#'   
+#' # using a model fitting workflow -----------------------
+#' 
+#' # fit a linear model predicting number of hours worked per
+#' # week using respondent age and degree status.
+#' observed_fit <- gss %>%
+#'   specify(hours ~ age + college) %>%
+#'   hypothesize(null = "independence") %>%
+#'   fit()
+#' 
+#' observed_fit
+#' 
+#' # fit 100 models to resamples of the gss dataset, where the response 
```

Comment:
Do comments in code examples always start with a small letter? Seems like should be sentence case if they'll end in a period, esp if there are multi-sentence comments. (If always like this throughout, nm.)

Reply:
The convention seems to be lowercase throughout since some examples have more brief commenting than others. A few places where this was inconsistent--just addressed. :-)